Fomod fail to install after CLI abstraction changes

Error:

```
00:00:24.649 [ERROR] Failed to install mod Immersive Citizens - AI Overhaul-173-0-4-0-2-1655813855|System.NullReferenceException: Object reference not set to an instance of an object.
   at NexusMods.CLI.CliOptionSelector.RenderOptions[TGroupId,TOptionId](IEnumerable`1 groups, IList`1 selectedGroup, String selectedGroupName) in C:\Dev\Repos\NexusMods.App\src\NexusMods.CLI\CliOptionSelector.cs:line 180
   at NexusMods.CLI.CliOptionSelector.RequestMultipleChoices[TGroupId,TOptionId](IEnumerable`1 groups) in C:\Dev\Repos\NexusMods.App\src\NexusMods.CLI\CliOptionSelector.cs:line 74
   at NexusMods.Games.FOMOD.CoreDelegates.UiDelegates.UpdateState(InstallerStep[] installSteps, Int32 currentStepId) in C:\Dev\Repos\NexusMods.App\src\Games\NexusMods.Games.FOMOD\CoreDelegates\UiDelegate.cs:line 77
   at FomodInstaller.Scripting.XmlScript.XmlScriptExecutor.sendState(IList`1 lstSteps, String strPrefixPath, Int32 stepIdx)
   at FomodInstaller.Scripting.XmlScript.XmlScriptExecutor.processStep(IList`1 lstSteps, Int32 stepIdx, TaskCompletionSource`1 Source, XmlScript xscScript, List`1 PluginsToActivate)
   at FomodInstaller.Scripting.XmlScript.XmlScriptExecutor.DoExecute(IScript scpScript, String dataPath, Object preset)
   at CallSite.Target(Closure, CallSite, Object)
   at System.Dynamic.UpdateDelegates.UpdateAndExecute1[T0,TRet](CallSite site, T0 arg0)
   at FomodInstaller.Scripting.ScriptExecutorBase.Execute(IScript p_scpScript, String p_strDataPath, Object preset)
   at NexusMods.Games.FOMOD.FomodXmlInstaller.GetModsAsync(GameInstallation gameInstallation, ModId baseModId, Hash srcArchiveHash, EntityDictionary`2 archiveFiles, CancellationToken cancellationToken) in C:\Dev\Repos\NexusMods.App\src\Games\NexusMods.Games.FOMOD\FomodXmlInstaller.cs:line 64
   at NexusMods.DataModel.ArchiveInstaller.AddMods(LoadoutId loadoutId, Hash archiveHash, String defaultModName, CancellationToken token) in C:\Dev\Repos\NexusMods.App\src\NexusMods.DataModel\ArchiveInstaller.cs:line 104
```

--- a/src/NexusMods.App/Program.cs
+++ b/src/NexusMods.App/Program.cs
@@ -6,6 +6,7 @@ using Avalonia;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
+using NexusMods.Abstractions.CLI;
 using NexusMods.App.Listeners;
 using NexusMods.App.UI;
 using NexusMods.CLI;
@@ -48,6 +49,12 @@ public class Program
                 .Build();
 
             return await builder.InvokeAsync(args);
+        }
+        else
+        {
+            var selector = host.Services.GetRequiredService<CliOptionSelector>();
+            var renderers = host.Services.GetServices<IRenderer>();
+            selector.Renderer = renderers.FirstOrDefault(r => r.Name == "console") ?? renderers.First();
         }
 
         // Start listeners only available in GUI mode

--- a/src/NexusMods.CLI/CommandLineConfigurator.cs
+++ b/src/NexusMods.CLI/CommandLineConfigurator.cs
@@ -35,6 +35,7 @@ public class CommandLineConfigurator
         _provider = provider;
         _verbs = verbs.ToArray();
         _getOptionMethod = typeof(CommandLineConfigurator).GetMethod(nameof(GetOption), BindingFlags.Instance | BindingFlags.NonPublic)!;
+        selector.Renderer = _defaultConsoleRenderer;
     }
 
     /// <summary>

